tests/10-migration-backend-tests/testcafe/migrations/media-audio.csv

```diff
@@ -1,2 +1,2 @@
 unique_id,name,access_terms,original_name,mime_type,media_of,media_use,url
-media_audio_00001,Moo Cow,:::media_accesscontrol_01||:::media_accesscontrol_02,moo.mp3,audio/mpeg,:::media_io_01,Original File||Preservation Master File,http://migration-assets/assets/audio/moo.mp3
+media_audio_00001,Moo Cow,:::media_accesscontrol_01||:::media_accesscontrol_02,moo.mp3,audio/mpeg,:::media_io_1,Original File||Preservation Master File,http://migration-assets/assets/audio/moo.mp3
```

Comment:
This change in reference is only needed because this PR change the value of the the `unique_id` in `media-islandora_object.csv`, so this change, while it was caught by validation (good!), wasn't an issue.